src/PSR7Session/Http/SessionMiddleware.php

```diff
@@ -94,7 +101,8 @@ public function __construct(
         SetCookie $defaultCookie,
         Parser $tokenParser,
         int $expirationTime,
-        int $refreshTime = self::DEFAULT_REFRESH_TIME
+        int $refreshTime = self::DEFAULT_REFRESH_TIME,
+        ClockInterface $clock = null
```

Comment:
That would be ideal. I can format the changelog myself post-merge, what it needs is just:
 1. a description of the breakage
 2. why the breakage was introduced
 3. instructions for how to upgrade